docs/source/writing_benchmarks.rst

```diff
@@ -279,6 +292,23 @@ decorator ``@skip_for_params`` as::
 This skips running benchmarks for these and also the setup functions. However,
 ``setup_cache`` is not affected.
 
+If the onus of preparing the exact parameter sets for ``skip_for_params`` is too
+complicated and the ``setup`` function is not too expensive, or if a benchmark
+needs to be skipped conditionally but ``skip_if`` is not the right choice, there
```

Comment:
Can `skip_if` be used? If so there should be an example.

Reply:
Sorry, had split the documentation across PRs.